### PR TITLE
inclusive terms

### PR DESCRIPTION
**Pull request by Braundo**
## Hey, I just made a Pull Request!

more inclusive terminology

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

Signed-off-by: Aaron Braundmeier <aaronbraundmeier@gmail.com>

### CODE REVIEW

File: packages/cli/config/eslint-factory.js [hunk 1]
```diff
@@ -231,7 +231,7 @@ function createConfigForRole(dir, role, extraConfig = {}) {
         },
         restrictedImports: [
           {
-            // Importing the entire MUI icons packages kills build performance as the list of icons is huge.
+            // Importing the entire MUI icons packages stops build performance as the list of icons is huge.
```
Comment (jamieklassen) [inline packages/cli/config/eslint-factory.js:234]:
```suggestion
            // Importing the entire MUI icons packages impedes build performance as the list of icons is huge.
```

**Review by agentbellnorm — APPROVED** [review]
This is great, thanks!

File: packages/dev-utils/src/devApp/render.tsx [hunk 2]
```diff
@@ -165,9 +165,9 @@ export class DevAppBuilder {
    * Build a DevApp component using the resources registered so far
    */
   build(): ComponentType<{}> {
-    const dummyRouteRef = createRouteRef({ id: 'dummy' });
-    const DummyPage = () => <Box p={3}>Page belonging to another plugin.</Box>;
-    attachComponentData(DummyPage, 'core.mountPoint', dummyRouteRef);
+    const sampleRouteRef = createRouteRef({ id: 'sample' });
+    const SamplePage = () => <Box p={3}>Page belonging to another plugin.</Box>;
+    attachComponentData(SamplePage, 'core.mountPoint', sampleRouteRef);
```
Comment (freben) [inline packages/dev-utils/src/devApp/render.tsx:170]: "fake" throughout here too

File: packages/techdocs-cli/e2e-tests/techdocs-cli.test.ts [hunk 3]
```diff
@@ -64,7 +64,7 @@ describe('end-to-end', () => {
 
   afterEach(async () => {
     // On Windows the pid of a spawned process may be wrong
-    // Because of this, we should be kill the MKDocs after the test
+    // Because of this, we should be stop the MKDocs after the test
```
Comment (freben) [inline packages/techdocs-cli/e2e-tests/techdocs-cli.test.ts:67]: maybe remove the "be" while we are at it

**Review by freben — COMMENTED** [review]
Thanks! Some adjustments suggested. Also, you will need to add the actual package names in the frontmatter of your changeset. Easiest may be to just run `yarn changeset` and let it suggest them for you.

**Review by Braundo — COMMENTED** [review]
Signed-off-by: Aaron Braundmeier [aaronbraundmeier@gmail.com](mailto:aaronbraundmeier@gmail.com)

**Review by Braundo — COMMENTED** [review]
Signed-off-by: Aaron Braundmeier [aaronbraundmeier@gmail.com](mailto:aaronbraundmeier@gmail.com)

**Review by Braundo — COMMENTED** [review]
Signed-off-by: Aaron Braundmeier [aaronbraundmeier@gmail.com]

**Comment by freben** [comment]
The conflict in packages/backend-common/src/context/RootContext.ts should be OK to revert (ie your PR no longer needs to change that file, I believe, since it's already fixed in master, so you can skip it)

**Comment by freben** [comment]
The Details link on the DCO build (the check itself in the bottom of the page) has some guidance on how to resolve the DCO warning. https://github.com/backstage/backstage/pull/14790/checks?check_run_id=9777491583

**Comment by freben** [comment]
@Braundo I helped rebase + squash + resolve the conflict on this one, force pushing to your branch. I hope you do not mind. I really hope to get this in now if things turn green.

**Review by freben — APPROVED** [review]

**Pull request merged**